.eslintrc

```diff
@@ -1,9 +1,6 @@
 {
-  "extends": "./index.js",
+	"extends": "./index.js",
 	"env": {
 		"node": true
-	},
-	"rules": {
-		"sort-keys": ["error", "asc"]
```

Comment:
As much as I like sorted, I think it makes more sense to be able to sort by context, and not just alphabetical

Reply:
Please revert this change - the reason its added is to ensure sensible diffs and reduce need for manual sorting. We do not need things sorted by context as these configs are modified on a per rule basis (usually).